test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,25 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // only one object
+        assertTrue(Utils.isAnyNull((Object)null));
+        assertFalse(Utils.isAnyNull(1));
+        assertFalse(Utils.isAnyNull("ah"));
+        assertFalse(Utils.isAnyNull(""));
+
+        // all null objects
+        assertTrue(Utils.isAnyNull((Object)null, (Object)null, (Object)null));
+
+        // some null objects
+        assertTrue(Utils.isAnyNull(1, 1, (Object)null));
+        assertTrue(Utils.isAnyNull((Object)null, 1, (Object)null));
+        assertTrue(Utils.isAnyNull((Object)null, "ah", (Object)null));
+        assertTrue(Utils.isAnyNull((Object)null, "", (Object)null));
```

Comment:
Sometimes when adding test cases, think about how the method treat this new test case differently from the other existing ones.